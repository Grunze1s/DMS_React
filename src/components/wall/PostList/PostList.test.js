import React from 'react';
import PostList from './index';
import {
    configure,
    shallow
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})


const setup = (props = {}, state = null) => {
    return (shallow( < PostList {...props}  />)
    )
}
const dummy_data={
    postList:[{id:2,  
        user:"user1", 
        date:"20 AUg, 2020",
         mediaList:[], viewCount:3, 
         likeCount:2, isLike:true, 
         commentList:[], text:"efdf"},
         {id:3,user:"user1", date:"20 AUg, 2020", mediaList:[],
          viewCount:3, likeCount:2, isLike:true, commentList:[], 
          text:"efdf"},{id:4,user:"user1", date:"20 AUg, 2020", 
          mediaList:[], viewCount:3, 
          likeCount:2, isLike:true, commentList:[], text:"efdf"}],
    user:"user1",
  
}
describe('<PostList />', () => {
  

it("PostList render", () => { 
        const wrapper= setup({...dummy_data});
        expect(wrapper).toMatchSnapshot();
  
})
})

