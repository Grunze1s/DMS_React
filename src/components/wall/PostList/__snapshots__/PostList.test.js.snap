// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostList /> PostList render 1`] = `ShallowWrapper {}`;
